src/store/modules/groupApplications.js

```diff
@@ -100,6 +103,9 @@ export default {
     update (state, application) {
       Vue.set(state.entries, application.id, application)
     },
+    clear (state) {
+      state.entries = {}
```

Comment:
We usually used the `initialState` function to clear the state of a module, for example here:
https://github.com/yunity/karrot-frontend/blob/1831610517ee3d7247c33df53999da8405f6eb42/src/store/modules/currentThread.js#L152

With this, we don't need to update the `clear` mutation when a new property gets added to initialState.